frameworks/kafka/universe/config.json

```diff
@@ -40,6 +40,11 @@
             "type": "boolean",
             "default": false
           },
+          "tls": {
+            "description": "Enable TLS",
```

Comment:
`Enable the TLS. This works only on Mesosphere DC/OS Enterprise installed in permissive or security modes.`